README.md

```diff
@@ -1,2 +1,15 @@
 # d-sparse-sets-ecs
-An improved version of my previous ecs-d **heavily** influeced by entt
+An improved version of my previous ecs-d **heavily** influeced by
+[EnTT](https://github.com/skypjack/entt/tree/2e891fdd7b442f1a56b1e821eda99b0e8a1ddbaf)
+
+# [EnTT](https://github.com/skypjack/entt/tree/2e891fdd7b442f1a56b1e821eda99b0e8a1ddbaf)
+`EnTT` is a header-only, tiny and easy to use library for game programming and
+much more written in modern C++, mainly known for its innovative
+entity-component-system (ECS) model.
+
+![EnTT: Gaming meets modern C++](https://user-images.githubusercontent.com/1812216/42513718-ee6e98d0-8457-11e8-9baf-8d83f61a3097.png)
+
+Check or follow this amazing project at
+https://user-images.githubusercontent.com/1812216/42513718-ee6e98d0-8457-11e8-9baf-8d83f61a3097.png
```

Comment:
You've linked to an image, probably not what you intended :stuck_out_tongue:

Reply:
Nice catch! Done